app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -74,4 +82,12 @@ class MainActivity : AppCompatActivity() {
         val navController = findNavController(R.id.nav_host_fragment_content_main)
         return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
     }
+
+    fun setAppBarTitle() {
+
+    }
+
+    fun showMenuItems(keybindsFragmentActionMenuIds: Any) {
+
+    }
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.